cpp/log_rotate.cc

```diff
@@ -117,6 +127,9 @@ int Main(int argc, char *argv[]) {
                                                FileUtil::GroupnameFromGID(gid));
                     if (SELinuxUtil::IsEnabled())
                         SELinuxUtil::FileContext::ApplyChanges(filename);
+
+                    for (const auto pid : pids)
+                        ::kill(pid, SIGHUP);
```

Comment:
Is it necessary to send SIGHUP also to ourself?
If not, it might be useful if ProcessUtil::GetProcessIdsForPath would take an additional parameter to include or exclude the process itself from the result.

Reply:
Fixed.